flow-shop/chapter.md

```diff
@@ -87,10 +89,9 @@ if __name__ == '__main__':
     print_solution(data, perm)
 ```
 
-We describe the parsing of Taillard problem files shorty, and the files themselves can be found at the following URL:
-* http://mistic.heig-vd.ch/taillard/problemes.dir/ordonnancement.dir/ordonnancement.html
+We will describe the parsing of Taillard problem files shortly, and the files themselves are [available online](http://mistic.heig-vd.ch/taillard/problemes.dir/ordonnancement.dir/ordonnancement.html).
```

Comment:
It will, but in the print book, URLs will be footnotes, not part of running text. In the ebooks they will be links.